conda/environments/ucxx-cuda118_arch-x86_64.yaml

```diff
@@ -8,9 +8,11 @@ dependencies:
   - cudatoolkit=11.8
   - pip
 # RAPIDS
-  - cudf=23.06
-  - dask-cudf=23.06
-  - dask-cuda=23.06
+  - cudf==23.4.*
+  - dask-cuda==23.4.*
+  - dask-cudf==23.4.*
+  - librmm==23.4.*
+  - rmm==23.4.*
```

Comment:
these should all be `23.6`, right?

Reply:
Yes! Thanks for catching. I did a big mess locally and screwed up when fixing, thankfully you noticed! 🙂

Reply:
It's fixed now.